functions-methods.js

```diff
@@ -22,12 +52,34 @@
 
 
 
-/* Opdracht  3 */
-// Schrijf een functie genaamd checkEmailValidity, die een emailadres verwacht en checkt of het emailadres valide is. De functie returned true of false, afhankelijk van de uitkomst.
-// Een emailadres is valide wanneer:
+/* Opdracht 3 */
+
+// Schrijf een functie genaamd checkEmailValidity, die een e-mailadres verwacht en checkt of het e-mailadres valide is.
+// De functie returned true of false, afhankelijk van de uitkomst.
+// Een e-mailadres is valide wanneer:
 // * Er een @ in voorkomt
-// * Er géén , in voorkomt
-// * Er géén . in voorkomt als allerlaatste karakter (dus hotmail.com is valide, net als outlook.nl, maar outlooknl. niet)
+// * Er géén "," in voorkomt
+// * Er géén "." in voorkomt als allerlaatste karakter (dus hotmail.com is valide, net als outlook.nl, maar outlooknl. niet)
+
+function checkEmailValidity(email) {
+    const check_for_at = email.includes("@");
+    const check_for_comma = email.includes(",");
+    const check_for_lastindex = email.endsWith(".");
+
+    if (check_for_at === false ||
+        check_for_comma === true ||
+        check_for_lastindex === true) {
+        console.log(false);
+    }else {
+        console.log(true);
+    }
```

Comment:
In principe prima, maar dit leest wat makkelijker en is minder code 😉 
```suggestion
    console.log(check_for_at && !check_for_comma && !check_for_lastindex);
```